### DISCUSSION TITLE
[Help] openpose/replicating a pose

### DISCUSSION THREAD

**Discussion by IroyzoI** [Mikubill/sd-webui-controlnet#1816] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I was trying to use controlnet to replicate a pose except with a prefered character lora, but i couldnt make it work, i looked for tutorials and tried those, nothing came close than just using img2img, is there a tutorial on this subject?

### Steps to reproduce the problem

What i did was, get the pose i wanted, put it in and used different Preprocessor's, them being anime lineart, open pose and reference, i couldnt get the reference pose from any of them, 

### What should have happened?

the exact specific pose ( https://imgur.com/a/enpPmEW ) except with the prefered character lora

### Commit where the problem happens

webui: 
controlnet: 

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
no
```

### List of enabled extensions

Lycoris
openpose editor by fkunn1326
controlnet
openpose editor by huchenlei
LDSR
Lora
ScuNET
swinIR
canvas zoom and pan
extra options section
prompt bracket checker

### Console logs

```Shell
no bug
```

**Comment by huchenlei** [comment]
Please provide more context. What LoRA are you using? The screenshot of your controlnet and txt2img setup. What are the generation results?

**Reply by IroyzoI** [reply]
Hello, the lora im using is this: https://civitai.com/models/83452/emilia-oror-hundred 
![00045-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/52869aa4-98c1-40a1-8f51-b889b96f9399)
![00037-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/69ee8618-9ff1-49e3-8611-4f4945acd74d)
![00038-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/3bca24eb-b049-484d-b82f-0106a24a1e78)
![00039-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/fef833f7-7746-4eba-a4a8-b69d1de3f9b9)
![00040-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/9e3be458-7c54-4bd8-96bb-b88dfa3e2491)
![00041-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/dc7bbd86-7f71-498b-a21e-72a470720cfc)
![00042-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/fc275de7-399f-4af9-9928-b244e77060ff)
![00043-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/b19daadb-ea3d-497c-a479-b03e70aa6935)
![00044-1812022313](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/b9ae5131-3dd5-4b58-a9d5-b0ab83205b4a)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/b8998279-ea4c-4fa8-97ac-6156abb693c0)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/8628d242-f856-4949-8ac0-7cd2e391ed3e)

**Reply by huchenlei** [reply]
Seems like you used the wrong control type. If you want to replicate the pose, you need to use `Openpose` control type instead of `Reference`.

**Reply by IroyzoI** [reply]
I just tried it with openpose option, and selected openpose_full preprocessor with control openpose v11p as model but it came out as this 
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/0293638e-d136-4f19-ad58-bc021e6ecb60)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/115279298/c9e09c3a-0ddc-4df7-8ffc-4682d1ac0a54)

**Reply by huchenlei** [reply]
Openpose detection is trained on realistic images so the algorithm does not work well with animie images. You can use [openpose editor](https://github.com/huchenlei/sd-webui-openpose-editor) to manually adjust keypoints to match you reference image. In your case the head of the character is not recognized by preprocessor.

**Reply by aartikov** [reply]
@IroyzoI try "ControlNet is more important" for openpose.
